opacus/tests/grad_samples/dp_lstm_test.py

```diff
@@ -6,6 +6,79 @@
 from opacus.layers import DPLSTM
 
 from .common import GradSampleHooks_test
+from torch.nn.utils.rnn import pad_sequence, pack_padded_sequence, PackedSequence
+
+from typing import List, Optional, Tuple, Union
+
+def _gen_packed_data(
+    minibatch_size: int,
+    max_seq_length: int,
+    input_dim: int,
+    batch_first: bool,
+    sorted_: Optional[bool] = False,
+) -> PackedSequence:
+    """
+    This is used to generate random PackedSequence data, sampled from a normal distribution, for testing DPLSTM.
+
+    Args:
+        minibatch_size : Total number of sequences to generate
+        max_seq_length : The maximum number of timesteps of a sequence
+        input_dim : The embedding dimension of a sequence at any timestep
+        batch_first : If this is true, data is first generated using a padded sequence of dimension (minibatch_size x max_seq_len x input_dim) , else: (max_seq_length x minibatch_size x input_dim)
+        sorted_ : If this is true then the original generated data used to produce the PackedSequence will already be ordered based on sequence lengths, else a random order and the 'sorted_indices'
+                    and 'unsorted_indices' fields will be None.
+
+    Return Value:
+        packed_data : A PackedSequence object with its data sampled from a normal distribution.
+    """
+
+    if batch_first:
+        data = []
+        seq_lengths = []
+        for i in range(minibatch_size):
```

Comment:
linter will complain about `i` not being used within the loop. Please use `for _ in range(minibatch_size)` instead